ksqldb-parser/src/main/java/io/confluent/ksql/util/QueryMask.java

```diff
@@ -57,11 +42,13 @@ private String getMaskedStatement(final String query) {
     }
   }
 
-  private String maskConnectProperties(final String query) {
+  private static String maskConnectProperties(final String query) {
     /*
       Failed to parse statement. Mask disallowed keys by string replacing. This is a best effort
       attempt. The assumption is that properties are in right format:
-      "(\"[^\"]+\")\\s*=\\s*('[^']*'|\"[^\"]*\")")
+      "(\"[^\"]+\")\\s*=\\s*('[^']*'|\"[^\"]*\")") i.e. it matches key wrapped in double quotes
+      equals value wrapped in double or single quotes. Key and value could be empty and there could
+      be spaces around equal sign.
```

Comment:
Haven't read the regex so maybe this is already covered: there can be whitespace (not just spaces but also newlines) surrounding the commas too. See here for an example connector statement: https://developer.confluent.io/tutorials/internet-of-things/confluent.html